nominees/the-eviction-lab.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "The Eviction Lab",
+  "aliases": [
+    "The Eviction Lab at Princeton University"
+  ],
+  "description": "The Eviction Lab at Princeton University creates data, interactive tools, and research to help neighbors and policymakers understand the eviction crisis.",
+  "website": "https://evictionlab.org/",
+  "license": [
+    {
+      "spdx": "ODC-By-1.0",
+      "licenseURL": "https://data-downloads.evictionlab.org/"
+    },
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/EvictionLab/eviction-maps/blob/development/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 1,
+      "evidenceText": "Eviction is both a cause and a consequence of poverty, and addressing the eviction crisis can help in efforts to reduce poverty."
+    },
+    {
+      "SDGNumber": 3,
+      "evidenceURL": "https://evictionlab.org/why-eviction-matters/"
```

Comment:
SDG 3 is `Ensure healthy lives and promote well-being for all at all ages`. The evidence URL is not clear on its connection, but I assume it is the connection to mental health and wellbeing. It would be helpful for this to be articulated for the DPG Registry. I would ask the submitter to clarify this.

Also noting that if there is data available to support target 3.4 ("By 2030, reduce by one third premature mortality from non-communicable diseases through prevention and treatment and promote mental health and well-being") and indicator 3.4.2 (Suicide mortality rate), that is a helpful connection to make to the SDG specifically.